### PR TITLE
Change README.md

### PR DESCRIPTION
**Pull request by zwyan0**
Added pg16 and updated command examples.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,7 +14,8 @@ different PostgreSQL server versions without introducing server version
 check code blocks.  Please choose a branch to match the PostgreSQL version
 you will be building pg_rman against.
 
-* master : branch for PostgreSQL 15 [![Test](https://github.com/ossc-db/pg_rman/actions/workflows/build.yml/badge.svg?branch=master&event=push)](https://github.com/ossc-db/pg_rman/actions/workflows/build.yml)
+* master : branch for PostgreSQL 16 [![Test](https://github.com/ossc-db/pg_rman/actions/workflows/build.yml/badge.svg?branch=master&event=push)](https://github.com/ossc-db/pg_rman/actions/workflows/build.yml)
```
Comment (mikecaat) [inline README.md:17]: for v17?

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -41,11 +43,11 @@ jobs:
       shell: bash -xe {0}
       run: |
         declare -A versions=(
-          ["master"]="15.1"
-          ["REL_15_STABLE"]="15.1"
-          ["REL_14_STABLE"]="14.6"
-          ["REL_13_STABLE"]="13.9"
-          ["REL_12_STABLE"]="12.13"
+          ["master"]="16.1"
+          ["REL_15_STABLE"]="15.5"
```
Comment (mikecaat) [inline .github/workflows/build.yml:47]: Do you need to add REL_16_STABLE?

**Review by mikecaat — COMMENTED** [review]
Thanks!

File: README.md (hunk 1 above)
Comment (zwyan0) [reply README.md:17]: Yes. I will modify 16 to 17.

File: .github/workflows/build.yml (hunk 2 above)
Comment (zwyan0) [reply .github/workflows/build.yml:47]: Looking at last year's history, I will create REL_16_STABLE after copyright and other modifications.

File: README.md [hunk 3]
```diff
@@ -14,7 +14,8 @@ different PostgreSQL server versions without introducing server version
 check code blocks.  Please choose a branch to match the PostgreSQL version
 you will be building pg_rman against.
 
-* master : branch for PostgreSQL 15 [![Test](https://github.com/ossc-db/pg_rman/actions/workflows/build.yml/badge.svg?branch=master&event=push)](https://github.com/ossc-db/pg_rman/actions/workflows/build.yml)
+* master : branch for PostgreSQL 17 [![Test](https://github.com/ossc-db/pg_rman/actions/workflows/build.yml/badge.svg?branch=master&event=push)](https://github.com/ossc-db/pg_rman/actions/workflows/build.yml)
+* REL_16_STABLE : branch for PostgreSQL 16 [![Test](https://github.com/ossc-db/pg_rman/actions/workflows/build.yml/badge.svg?branch=REL_15_STABLE&event=push)](https://github.com/ossc-db/pg_rman/actions/workflows/build.yml)
```
Comment (zwyan0) [inline README.md:18]: At present, REL_16_STABLE does not exist, so leave it at 15 on the link below.

Comment (mikecaat) [reply README.md:18]: I think we will forget to change it. Since you will make the branch soon, I think it's better to change for v16

**Review by mikecaat — APPROVED** [review]

**Pull request merged**